packages/xo-vmdk-to-vhd/src/index.js

```diff
@@ -5,18 +5,20 @@ import VMDKDirectParser from './vmdk-read'
 
 export {
   default as readVmdkGrainTable,
-  readCapacityAndGrainTable,
+  readCapacityAndGrainTable
```

Comment:
Unrelated changes.

Makes sure your branch is up-to-date and the dependencies are installed, and do:

```
cd xen-orchestra
prettier --write '**/*.js'
```